src/System.Private.CoreLib/shared/System/Threading/Tasks/Task.cs

```diff
@@ -882,21 +886,17 @@ internal bool MarkStarted()
             return AtomicStateUpdate(TASK_STATE_STARTED, TASK_STATE_CANCELED | TASK_STATE_STARTED);
         }
 
-        internal bool FireTaskScheduledIfNeeded(TaskScheduler ts)
+        internal void EtwTaskScheduled(TaskScheduler ts)
```

Comment:
The original name for this (`FireTaskScheduledIfNeeded`) looked better.

Reply:
ok